app/events/db/migrations/db_table_sql_scripts.sql

```diff
@@ -711,5 +711,51 @@ CREATE TABLE IF NOT EXISTS iiq_schema.async_profile_analytics
 
     CONSTRAINT unique_async_profile_analytics UNIQUE (iiq_id)
 );
-
 ALTER TABLE iiq_schema.async_profile_analytics OWNER TO iiq;
+
+
+CREATE TABLE IF NOT EXISTS iiq_schema.async_contents
+(
+    id                     UUID      DEFAULT uuid_generate_v4() NOT NULL PRIMARY KEY,
+    status                 VARCHAR(50),
+    work_platform_id       UUID,
+    work_platform_name     VARCHAR(255),
+    work_platform_logo_url VARCHAR(255),
+    platform_username      VARCHAR(255),
+    profile_url            VARCHAR(255),
+    external_id            VARCHAR(255),
```

Comment:
is this iiq_id?

Reply:
No, this containts image URL of the work platform logo.
Eg: https://insightiq.ai/storage/linkedin.png